packages/ui/src/ui/utils/system/masters.ts

```diff
@@ -0,0 +1,5 @@
+import {calcShortNameByRegExp} from '../../containers/Host/Host';
+
+export const calcMasterShortName = (address: string) => {
```

Comment:
The function looks unnecessary, `address` value might be returned from `calcShortNameByRegExp`